tests/integration_tests/test_update_dashboard_task.py

```diff
@@ -49,7 +49,7 @@ def mock_get_dataframe(self, storage_name):
             if storage_name == UpdateDashboardTask.storage_hospitalizations:
                 return pd.read_csv(
                     os.path.join(THIS_DIR, '../samples/Dashboard PDF Development - hospitalization.csv'))
-            raise ValueError(f'Did not expect {storage_name=}')
```

Comment:
again an issue with the old python version you are using ...

Reply:
Updated it.